frontend/app/src/components/calendar/CalendarWeekdays.vue

```diff
@@ -0,0 +1,17 @@
+<script setup lang="ts">
+import Fragment from '@/components/helper/Fragment';
+
+const weekdays = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'];
```

Comment:
Should we hardcode them? can't we somehow use the locale/language and get the localized names?